pyscal/utils/relperm.py

```diff
@@ -11,34 +11,46 @@
 logger = logging.getLogger(__name__)
 
 
-def crosspoint(dframe: pd.DataFrame, satcol: str, kr1col: str, kr2col: str) -> float:
+def crosspoint(
+    inputframe: pd.DataFrame, satcol: str, kr1col: str, kr2col: str
+) -> float:
     """Locate the crosspoint where kr1col == kr2col
 
     Args:
-        dframe: Dataframe with at least three columns
+        inputframe: Dataframe with at least three columns
```

Comment:
Is it worth breaking api for a change from `dframe` to `inputframe`? At least it doesn't feel natural to do it in a PR on increased test coverage?

Reply:
Its not worth breaking the API for this, but is this an API break? This is not a keyword argument? The reason for change was to make it more clear that we are not trying to modify the incoming dataframe.

Reply:
(ping)

Reply:
wouldn't this break for someone that has used the `crosspoint` function in the following way?
```
some_float = crosspoint(dframe=some_df, satcol=some_col1, kr1col=some_col2, kr2col=some_col3)
``` 
Maybe it is a bad habit, but at least I like to write the argument name even though they are not required, as I find it easier to follow the code if I have an idea of what the input I give correspond to in the function I use.

Reply:
You're right, bad habit or not, this would break. Will change.

Reply:
Fixed